DataStructures.Tests/SegmentTrees/SegmentTreeUpdateTest.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using DataStructures.SegmentTrees;
+using NUnit.Framework;
+
+namespace DataStructures.Tests.SegmentTrees
+{
+    [TestFixture]
+    public class SegmentTreeUpdateTests
+    {
+        private SegmentTreeUpdate testTree = new SegmentTreeUpdate(new int[] {8, 9, 1, 4, 8, 7, 2});
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6278042)